content/blog/2024-02-06.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "2024-02-06"
+date = 2024-02-06
++++
+
+---
+
+Today is Tuesday, day 037 of 366 (10.10%). We're in week 06 of 52 (11.53%).
+
+## Planning
+
+What I am going to do today:
+
+- [x] use infinite scrolling to load all pages from the API
+
+## Activities executed
+
+Today I did a lot of research on infinite scrolling, but I wasn't really understantding it, so I tried to take the concept and apply it ina more simple way to see if that would help and it did. I was able to load snippets from other pages, but the number of pages is fixed, because I couldn't find a way to check how many pages the REST framework has. I think I'll just have to use a while loop to keep going to the next page until there is a 404 error.
```

Comment:
```suggestion
Today I did a lot of research on infinite scrolling, but I wasn't really understantding it, so I tried to take the concept and apply it in a simpler way to see if that would help and it did. I was able to load snippets from other pages, but the number of pages is fixed, because I couldn't find a way to check how many pages the REST framework has. I think I'll just have to use a while loop to keep going to the next page until there is a 404 error.
```